cdc/scheduler/internal/scheduler.go

```diff
@@ -42,6 +43,10 @@ type Scheduler interface {
 		currentTables []model.TableID,
 		// All captures that are alive according to the latest Etcd states.
 		aliveCaptures map[model.CaptureID]*model.CaptureInfo,
+		// barrier contains the barrierTs of those tables that have
+		// ddl jobs that need to be replicated. The Scheduler will
+		// broadcast the barrierTs to all captures through the Heartbeat.
```

Comment:
Could you add these comments to the proto file too?

Reply:
done